### PR TITLE
Small tweaks to conform with spec

### PR DESCRIPTION
**Pull request by luhring**
Signed-off-by: Dan Luhring <dluhring@chainguard.dev>

### CODE REVIEW

File: pkg/vex/vex.go [hunk 1]
```diff
@@ -58,12 +58,19 @@ type Metadata struct {
...
-}
 
-// VulnerabilityReference captures other identifier assigned to the CVE.
-type VulnerabilityReference struct {
-	RefType   string `json:"type"` // URL, OSV, FEDORA, etc
-	Reference string `reference:"ref"`
+	// Version is the document version. It must be incremented when any content
+	// within the VEX document changes, including any VEX statements included within
+	// the VEX document.
+	Version string `json:"version"`
```
Comment (luhring) [inline pkg/vex/vex.go:65]: @puerco Would it make more sense for this field's type to be `int` instead?

File: pkg/vex/vex.go [hunk 2]
```diff
@@ -58,12 +58,19 @@ type Metadata struct {
...
+
+	// Tooling expresses how the VEX document and contained VEX statements were
+	// generated. It's optional. It may specify tools or automated processes used in
+	// the document or statement generation.
+	Tooling string `json:"tooling,omitempty"`
+
+	// Supplier is an optional field.
```
Comment (luhring) [inline pkg/vex/vex.go:72]: Not sure what else to say here, there's really nothing about the field in the spec... 🤔

File: pkg/vex/vex.go (hunk 1 above)
Comment (puerco) [reply pkg/vex/vex.go:65]: No, that would exclude out a lot of versioning schemes like semver, etc

File: pkg/vex/vex.go (hunk 2 above)
Comment (puerco) [reply pkg/vex/vex.go:72]: I think its fine. once the group decides on the faith of this field we can improve the docs.

**Review by puerco — APPROVED** [review]

**Pull request merged**